core-bundle/assets/backend.js

```diff
@@ -30,7 +30,12 @@ application.register('contao--webauthn', WebAuthn);
 
 // Cancel all prefetch requests that contain a request token
 document.documentElement.addEventListener('turbo:before-prefetch', e => {
-    if ((new URLSearchParams(e.target.href)).has('rt') || e.target.classList.contains('header_back') || e.target.closest('.sf-toolbar') !== null || e.target.matches('[onclick^="Backend.openModalIframe("]')) {
+    if (
+        e.target.classList.contains('header_back')
+        || e.target.matches('[onclick^="Backend.openModalIframe("]')
+        || e.target.closest('.sf-toolbar') !== null
+        || (new URLSearchParams(e.target.href)).has('rt')
```

Comment:
```suggestion
        || (new URLSearchParams(e.target.search || '')).has('rt')
```
As `URLSearchParams` expects only the query string, not the whole URL.

Reply:
Is `.href` really an URL object already here and not a string?

Reply:
Because as far as I tested `.search` would reference `String.prototype.search` here.

Reply:
See #8101

Reply:
See:

![image](https://github.com/user-attachments/assets/6503e6a0-1d10-4696-9fe7-6350d13f9900)

Reply:
> Is `.href` really an URL object already here and not a string?

`.href` is always a string, but e.target is an object that has `.href` and `.search` see https://developer.mozilla.org/en-US/docs/Web/API/HTMLAnchorElement/search

Reply:
Yes, sorry, misread the change :)